src/common/const.go

```diff
@@ -220,6 +220,9 @@ const (
 	// ScannerSkipUpdatePullTime
 	ScannerSkipUpdatePullTime = "scanner_skip_update_pulltime"
 
+	// AuditLogDisable
+	AuditLogDisable = "audit_log_disable"
```

Comment:
Can we name it `AuditLogEvents`? If it's empty, all checkboxes in the UI would appear unchecked. If it contains values, the UI can render the checkboxes accordingly based on those values.

Reply:
+1